exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 45;
+var numberOfMentors = 10;
+var TotalOfAll = numberOfMentors + numberOfStudents;
+console.log(TotalOfAll);
```

Comment:
Your code is correctly adding 2 numbers and outputting the result. If you read the question and expected output, you will see that you should be using different numbers and outputting more details. Also, variable names in Javascript are normally in lower camel case, i.e. totalOfAll